packages/blade/android/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,25 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  package="com.blade">
```

Comment:
We need to change the package name to `com.razorpay.blade` if we want to send this app as an update to the existing app on the play store & app store
```suggestion
  package="com.razorpay.blade">
```